src/app/(homepage)/page.tsx

```diff
@@ -22,7 +23,7 @@ export default async function Home() {
       <Features />
       <PastEvents />
       <Organizers />
-      <Partners />
+      <Partners PartnersSlugs={GLOBALPARTNERSSLUGS} />
```

Comment:
First letter of props are always in lowercase

```suggestion
      <Partners partnersSlugs={GLOBALPARTNERSSLUGS} />
```